tendermint/src/trust_threshold.rs

```diff
@@ -22,16 +24,22 @@ pub trait TrustThreshold: Copy + Clone + Debug + Serialize + DeserializeOwned {
 /// Some clients might require more than +1/3 and can implement their own
 /// [`TrustThreshold`] which can be passed into all relevant methods.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(
+    try_from = "RawTrustThresholdFraction",
+    into = "RawTrustThresholdFraction"
+)]
 pub struct TrustThresholdFraction {
-    /// Numerator of the trust threshold fraction
-    #[serde(with = "serializers::from_str")]
```

Comment:
Do we know why we are serializing the numerator and denominator as strings?

Reply:
They're `u64`, which can only fully be represented in JSON by a string afaik.